# CHEM 223 - Organic Chemistry 3 Lab


## Foreword

This notebook contains calculations for my third and final quarter of organic chemistry lab. At the time of writing, it's April 2020 and deep in quarantine, so this lab is online. I know what you're thinking, an online ochem lab? My days of broken glassware, fudged data, and poorly taken IR and proton NMR spectra are over. For the past two quarters, organic chemistry labs have been a welcome relief from the rest of my reading-on-a-computer typing-on-a-computer staring-at-a-computer coursework. This was where the mild pain of organic chemistry was supposed to come together in a tangible and rewarding way. Guess not. I won't miss stressing about finishing reactions and cleaning up under time pressure, though.

In [106]:
class Reagent:
    def __init__(self, name, grams=None, moles=None, mmoles=None, mL=None, mw=None, density=None, M=None, pKa=None): # g/mL
        self.name = name
        self.grams = grams
        self.moles = moles
        self.mmoles = mmoles
        self.mL = mL
        self.mw = mw
        self.density = density
        self.M = M
        self.pKa = pKa
        
        for i in range(20): # for dependent measurements
            self.grams = self._calc_grams()
            self.moles = self._calc_moles()
            self.mmoles = self._calc_mmoles()
            self.mL = self._calc_mL()
            
    
    def _calc_grams(self):
        if self.grams:
            return float(self.grams)
        elif self.moles and self.mw:
            return float(self.moles) * float(self.mw)
        elif self.mmoles and self.mw:
            return float(self.mmoles) / 1000.0 / float(self.mw)
        elif self.mL and self.density:
            return float(self.mL) * float(self.density)
        return None
    
    def _calc_moles(self):
        if self.moles:
            return float(self.moles)
        elif self.mmoles:
            return float(self.mmoles) / 1000.0
        elif self.grams and self.mw:
            return float(self.grams) / float(self.mw)
        elif self.mL and self.M:
            return float(self.mL) / 1000.0 * self.M
        return None
    
    def _calc_mmoles(self):
        if self.mmoles:
            return float(self.mmoles)
        elif self._calc_moles():
            return self._calc_moles() * 1000.0
        return None
    
    def _calc_mL(self):
        if self.mL:
            return self.mL
        elif self.grams and self.density:
            return float(self.grams) / float(self.density)
        elif self.moles and self.mw and self.density:
            return float(self.moles) * float(self.mw) / float(self.d)
        elif self.mmoles and self.mw and self.density:
            return float(self.mmoles) * float(self.mw) / float(self.d) / 1000.0
        return None

## EXPERIMENT 1: SYNTHESIS OF METHYL SALICYLATE: A FISCHER ESTERIFICATION

### Reagents

In [116]:
sa = Reagent("salicylic acid", grams=9.7, mw=138.121)
meoh = Reagent("MeOH", mL=25.0, mw=32.04, density=0.792, pKa=15.54)
h2so4 = Reagent("Conc. H2SO4", mL=4.0, M=18.0, pKa=-2.8)
quiz_incorrect_h2so4 = Reagent("Incorrect amount pure H2SO4", mL=1.0, density=1.83, mw=98.07, pKa=-2.8)

### Equivalents

In [117]:
import pandas as pd

reagents = [sa, meoh, h2so4, quiz_incorrect_h2so4]
mmoles = [[r.name, r.mmoles] for r in reagents]
df = pd.DataFrame(mmoles, columns=["name", "equivalents"])
df['equivalents'] = df['equivalents'] / sa.mmoles
df

name  equivalents
0               salicylic acid     1.000000
1                         MeOH     8.799554
2                  Conc. H2SO4     1.025228
3  Incorrect amount pure H2SO4     0.265707

### Theoretical Yield

1:1 molar ratio between salicylic acid reagent and methyl salicate product

In [109]:
ms = Reagent("methyl salicylate", moles=sa.moles, mw=152.1494)
print(f"{ms.grams:.2f} g methyl salicylate")

10.69 g methyl salicylate


### Percent yield

In [119]:
py = 7.4 / ms.grams * 100
print(f"percent yield: {py:.2f}. Nice.")

percent yield: 69.25. Nice.


### Properties and uses of salicylic acid and methyl salicylate

Salicylic acid is a whitish-tan solid at room temperature. It can be used to remove the outer layer skin and is often found in skincare and acne products. Salicylic acid is also an active metabolite of the drug aspirin and is responsible for the anti inflammatory properties of aspirin.

Methyl salicylate, aka wintergreen oil, is a colorless liquid at room temperature with a minty or fruity odor. It is commonly used in fragrances, foods, and beverages. It is also used in traditional medicine.


### Procedure in lab manual vs youtube video

[youtube video](https://www.youtube.com/watch?v=lJLP2bcXDqY)

The lab manual procedure used different and smaller quantities of reagents. No boiling stones in the video. Reaction mixture in the video was refluxed for twice as long. The video used a rotovap immediately after refluxing to remove some of the excess methanol. In the video, only cold water was added to the sep funnel. The lab manual calls for cold water, EtOAc, and brine to be added to the sep funnel. The video showed an extra final wash step. When the rest of the extraction was completed, the lab manual procedure called for rotovapping off the EtOAc. The video mentioned but did not show the use of MgSO4 to remove excess water from the product.


### Percent yield

69.25%


### Rf/Polarity of methyl salicylate vs salicylic acid

Salicylic acid would have a lower Rf value because the carboxylic acid group in salicylic acid is more polar than the ester group in methyl salicylate.


### IR Spectrum of methyl salicylate vs 

Methyl salicylate IR spectrum has a peak at 1650 cm-1 from ester C=O stretch. There are also C-O stretches in the fingerprint region from 1000 - 1300 cm-1 characteristic of an ester, which are absent in the IR spectra of salicylic acid. The IR spectra of both methyl salicylate and salicylic have O-H stretches around 3000 to 3500 cm-1.


### HNMR of methyl salicylate

The signal at 10.5 is from the phenol H. This is not a good signal to base the integrations off because it is broad and difficult to integrate accurately. This means the other integrations are not directly indicitave of the number of protons causing the signals. The 4 aromatic protons cause the peaks between 6.7 and 7.9, but overlap and are difficult to distinguish from each other. The singlet at 3.9 is caused by the 3 protons in the methyl group on the end of the ester. It would be better to integrate the signals based off of this signal, using a value of 3, because it is strong and distinct.


### 13C NMR of methyl salicylate

The C in the ester is the furthest signal downfield. The second furthest downfield signal is likely the C attached to the OH.


### Most interesting thing I learned

Since salicylic acid is the principle metabolite of aspirin, I enjoyed reading about how aspirin is absorbed by the body and metabolized into salicylic acid.